src/cryoemservices/cli/clem_align_and_merge.py

```diff
@@ -26,52 +29,99 @@ def run():
         default=None,
         type=str,
-        help="Full file path to the metadata file associated with this dataset",
+        help=(
+            "Full file path to the metadata file associated with this dataset. If no "
+            "metadata file is provided, the function will attempt to find the matching "
+            "metadata file."
+        ),
     )
     # Align image stacks before flattening
     parser.add_argument(
         "--pre_align_stack",
         default=None,
         type=str,
-        help="Choose whether to align the image stacks before flattening, and how",
+        help=(
+            "NOT IMPLEMENTED YET. \n"
+            "Choose whether to align the image stacks before flattening, and how. \n"
+            "DEFAULT: 'null'"
+        ),
     )
     # Determine how the image is flattened
     parser.add_argument(
         "--flatten",
         default="mean",
         type=str,
-        help="Choose whether to flatten the image stacks, and how",
+        help=(
+            "Choose whether to flatten the image stacks, and how. \n"
+            "DEFAULT: 'mean' \n"
+            "VALUES: ['null', 'min', and 'max']"
+        ),
     )
     # Determine what image registration protocol to implement
     parser.add_argument(
         "--align_stacks",
         default=None,
         type=str,
-        help="Choose whether to align the image stacks before merging, and how",
+        help=(
+            "NOT IMPLEMENTED YET. \n"
+            "Choose whether to align the image stacks before merging, and how. \n"
+            "DEFAULT: 'null'"
+        ),
     )
     # Parse the arguments
     args = parser.parse_args()
 
-    # Resolve the arguments internally to pass on to the function
+    """
+    Parse arguments
+    """
     # Resolve image stacks parameter
+    # Validate input
+    images_arg: list[str] = args.images
     if any((not isinstance(file, str) for file in args.images)):
         raise TypeError(
             "One or more of the image files provided are of an invalid type"
         )
-    image_files = [Path(file) for file in args.images if Path(file).exists()]
+    # Check if a stringified list has been provided (for submission in recipes)
+    if (
+        len(images_arg) == 1
+        and images_arg[0].startswith(("['", '["'))
+        and images_arg[0].endswith(("']", '"]'))
+    ):
+        try:
+            image_files: list[Path] = [
+                Path(file)
+                for file in literal_eval(images_arg[0])
+                if Path(file).exists()
+            ]
+        except Exception:
+            raise Exception("Unable to evaluate stringified list")
+    # Evaluate normally as list of strings otherwise
+    else:
+        image_files = [Path(file) for file in images_arg if Path(file).exists()]
+    # Check that files have been found correctly
     if len(image_files) == 0:
         raise FileNotFoundError("No valid file paths provided")
 
+    # File list debugging
```

Comment:
You could possibly add a --debug flag rather than leaving some commented out lines